CDCgov/prime-reportstream
# HL7 ORC Mapping for OML messages

This PR adds mapping to the HL7 `ORC` field for the `OML` message type.

Test Steps:
1. Download the input OML sample file attached to this PR: [input.txt](https://github.com/CDCgov/prime-reportstream/files/12489938/input.txt), and rename it with a `json` or `fhir` extension
2. Run `./gradlew primeCLI --args='fhirdata --output-format HL7 --input-file "/path/to/input.json"  -s "./metadata/hl7_mapping/receivers/Flexion/TILabOrder.yml"'`
3. Check to see that the HL7 message has the `ORC` field. Subfields to look for are `ORC-2` and `ORC-21`, the rest were already mapped by RS.

**Note**: we'll create a separate PR to add a sample OML file to the integration tests

## Changes
- Added mapping for `ORC-2` and `ORC-21` in `prime-router/metadata/hl7_mapping/OML_O21/base/service-request-order.yml`

## Checklist

### Testing

## Linked Issues
[Create Outbound OML ORC Segment](https://github.com/CDCgov/trusted-intermediary/issues/492)
